nativelink-service/tests/ac_server_test.rs

```diff
@@ -48,29 +48,28 @@ async fn insert_into_store<T: Message>(
     let data_len = store_data.len();
     let digest = DigestInfo::try_new(hash, action_size)?;
     store.update_oneshot(digest, store_data.freeze()).await?;
+
     Ok(data_len.try_into().unwrap())
 }
 
 async fn make_store_manager() -> Result<Arc<StoreManager>, Error> {
     let store_manager = Arc::new(StoreManager::new());
-    store_manager.add_store(
+    make_and_add_store_to_manager(
```

Comment:
For my understanding, what is the reason why you moved away from `store_factory` rather than modifying `store_factory`?

Reply:
Still using `store_factory` to build the Store, but wrapping up in `make_and_add_store_to_manager` because it makes building and adding a Store to a StoreManager a single step from a public-facing perspective.